src/coreclr/pal/src/include/pal/context.h

```diff
@@ -60,6 +60,90 @@ using asm_sigcontext::_xstate;
 bool Xstate_IsAvx512Supported();
 #endif // XSTATE_SUPPORTED || (HOST_AMD64 && HAVE_MACH_EXCEPTIONS)
 
+#if defined(HOST_64BIT) && defined(HOST_ARM64) && !defined(TARGET_FREEBSD) && !defined(TARGET_OSX)
+#if !defined(SVE_MAGIC)
```

Comment:
Is this define not present when building in our CI?

Reply:
> Is this define not present when building in our CI?

Yes, they are missing in the CI. If I remove any of them the build falls over. I think this is when cross compiling. Eithe way, it must be an old Linux being used because these defines have been present in Linux since about 2017.